docs/source/onedal/algorithms/svm/svm.rst

```diff
@@ -85,6 +93,38 @@ for reduced to the solution of the quadratic optimization problem
 
     Where :math:`\epsilon` is the error tolerance parameter.
 
+  .. group-tab:: Nu-classification
+
+    .. math::
+          \underset{\alpha }{\mathrm{min}}\frac{1}{2}{\alpha }^{T}Q\alpha
+
+    with :math:`0 \leq \alpha_i \leq \frac{1}{n}`, :math:`i = 1, \ldots, n`,
+    :math:`e^T\alpha \geq \nu`, :math:`y^T\alpha = 0`,
+    where :math:`e` is the vector of ones, :math:`\nu` is an upper bound on the fraction
+    of training errors and a lower bound of the fraction of support vector,
+    :math:`Q` is a symmetric matrix of size :math:`n \times n` with :math:`Q_{ij} = y_i y_j K(x_i, x_j)`,
+    and :math:`K(x,y)` is a kernel function.
+
+  .. group-tab:: Nu-regression
+
+    .. math::
+          \underset{\alpha }{\mathrm{min}}\frac{1}{2}{\alpha }^{T}Q\alpha
```

Comment:
not correct formula 
![image](https://user-images.githubusercontent.com/17429885/120785647-fd631600-c535-11eb-854b-510a3cc9da84.png)